sal_interfaces/MTM2/MTM2_Commands.xml

```diff
@@ -1,133 +1,103 @@
+        <Units>newton</Units>
+        <Count>6</Count>
     </item>
 </SALCommand>
 
 <SALCommand>
- <Subsystem>MTM2</Subsystem>
- <Version>3.8.0</Version>
- <Author>salgenerator</Author>
- <EFDB_Topic>MTM2_command_applyForce</EFDB_Topic>
- <Alias>applyForce</Alias>
- <Device>support</Device>
- <Property>actuators</Property>
- <Action></Action>
- <Value></Value>
- <Description>Provides the delta force vector used in adjusting the target force settings.</Description>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_positionMirror</EFDB_Topic>
+    <Alias>positionMirror</Alias>
+    <Description>Adjust the mirror's rigid body positions based on LTS-136.</Description>
     <item>
-      <EFDB_Name>forceSetPoint</EFDB_Name>
-      <Description>Force in newtons, increasing force corresponds to increasing tension on the load cell.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>newton</Units>
-      <Count>72</Count>
+        <EFDB_Name>x</EFDB_Name>
+        <Description>Position x.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>micron</Units>
+        <Count>1</Count>
     </item>
-</SALCommand>
-
-<SALCommand>
- <Subsystem>MTM2</Subsystem>
- <Version>3.8.0</Version>
- <Author>salgenerator</Author>
- <EFDB_Topic>MTM2_command_setCorrectionMode</EFDB_Topic>
- <Alias>setCorrectionMode</Alias>
- <Device>support</Device>
- <Property>actuators</Property>
- <Action></Action>
- <Value></Value>
- <Description>Selects between using the delta force updates or the bending modes in adjusting the target force settings.</Description>
     <item>
-      <EFDB_Name>mode</EFDB_Name>
-      <Description>An enumeration: 1 = BendingMode, 2 = Force.</Description>
-       <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
+        <EFDB_Name>y</EFDB_Name>
+        <Description>Position y.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>micron</Units>
+        <Count>1</Count>
     </item>
-</SALCommand>
-
-<SALCommand>
- <Subsystem>MTM2</Subsystem>
- <Version>3.8.0</Version>
- <Author>salgenerator</Author>
- <EFDB_Topic>MTM2_command_positionMirror</EFDB_Topic>
- <Alias>positionMirror</Alias>
- <Device>support</Device>
- <Property>actuators</Property>
- <Action></Action>
- <Value></Value>
- <Description>Adjust the mirror's rigid body positions: tip, tilt, and piston.</Description>
     <item>
-      <EFDB_Name>xTilt</EFDB_Name>
-      <Description>Tilt x in xy plane.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>radian</Units>
-      <Count>1</Count>
+        <EFDB_Name>z</EFDB_Name>
+        <Description>Position z.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>micron</Units>
+        <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>yTilt</EFDB_Name>
-      <Description>Tilt y in xy plane.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>radian</Units>
-      <Count>1</Count>
+        <EFDB_Name>xTilt</EFDB_Name>
```

Comment:
I would rather call this rotX, rotY and rotZ (or xRot,..). Tip/tilt are a bit misleading terms. LTS-136 speaks about rotations - no mention of tilt (or tip).

Reply:
Fixed in commit: 6af5a7b. Thanks!